sql/core/src/main/scala/org/apache/spark/sql/execution/limit.scala

```diff
@@ -358,7 +358,9 @@ case class TakeOrderedAndProjectExec(
     val orderByString = truncatedString(sortOrder, "[", ",", "]", maxFields)
     val outputString = truncatedString(output, "[", ",", "]", maxFields)
 
-    s"TakeOrderedAndProject(limit=$limit, orderBy=$orderByString, output=$outputString)"
+    val offsetStr = if (offset > 0) { s" offset=$offset, " } else { "" }
```

Comment:
I assume we only need to skip when offset = 0?

Reply:
> I assume we only need to skip when offset = 0?

@amaliujia Thank you for your review. I made some modifications.